CHANGELOG.md

```diff
@@ -82,6 +82,7 @@
 ## v119 (2018-02-28)
 
 - Install and prune devDependencies by default (#519)
+- Split `NPM_CONFIG_PRODUCTION` into `NPM_CONFIG_PRODUCTION` and `YARN_PRODUCTION` (for npm and Yarn respectively) (#519)
```

Comment:
I think this is more accurately phrased as:

No longer `export NPM_CONFIG_PRODUCTION=true` by default

Reply:
Re. https://github.com/heroku/heroku-buildpack-nodejs/pull/633#issuecomment-470220897, I was trying to communicate that potential usage of `NPM_CONFIG_PRODUCTION=false` would have to be replaced with `YARN_PRODUCTION=false`.